### PR TITLE
Release v1.1.1

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Release for Marketplace

- [x] Requires https://github.com/mattermost/mattermost-plugin-bitbucket/pull/45 to be merged first

#### Ticket Link
Part of https://github.com/mattermost/mattermost-plugin-bitbucket/issues/46

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -4,60 +4,73 @@
...
-          "linux-amd64": "server/dist/plugin-linux-amd64",
-          "darwin-amd64": "server/dist/plugin-darwin-amd64",
-          "windows-amd64": "server/dist/plugin-windows-amd64.exe"
-      }
+        "executables": {
+            "linux-amd64": "server/dist/plugin-linux-amd64",
+            "darwin-amd64": "server/dist/plugin-darwin-amd64",
+            "windows-amd64": "server/dist/plugin-windows-amd64.exe"
+        },
+        "executable": ""
```
Comment (larkox) [inline plugin.json:17]: Probably nothing to worry about, but I found strange this change on the extructure.

**Review by larkox — APPROVED** [review]
LGTM, just one small comment.

File: plugin.json [hunk 2]
```diff
@@ -4,60 +4,73 @@
...
     },
     "webapp": {
         "bundle_path": "webapp/dist/main.js"
     },
     "settings_schema": {
         "header": "To set up the Bitbucket plugin, you need to register a Bitbucket OAuth consumer here https://bitbucket.org/YOURWORKSPACE/workspace/settings/oauth-consumers/new.",
+        "footer": "* To report an issue, make a suggestion or a contribution, [check the repository](https://github.com/kosgrz/mattermost-plugin-bitbucket).",
```
Comment (jfrerich) [inline plugin.json]:
```suggestion
        "footer": "* To report an issue, make a suggestion or a contribution, [check the repository](https://github.com/mattermost/mattermost-plugin-bitbucket).",
```

File: plugin.json [hunk 3]
```diff
@@ -4,60 +4,73 @@
...
                 "key": "EnablePrivateRepo",
                 "display_name": "Enable Private Repositories",
                 "type": "bool",
-                "help_text": "(Optional) Allow the plugin to work with private repositories. Enabling private repositories will require existing users to reconnect their accounts to gain access to private repositories. A message will be automatically be sent to affected users next time they load Mattermost alerting them of this."
+                "help_text": "(Optional) Allow the plugin to work with private repositories. Enabling private repositories will require existing users to reconnect their accounts to gain access to private repositories. A message will be automatically be sent to affected users next time they load Mattermost alerting them of this.",
+                "placeholder": "",
+                "default": null
             }
-        ],
-        "footer": "* To report an issue, make a suggestion or a contribution, [check the repository](https://github.com/kosgrz/mattermost-plugin-bitbucket)."
```
Comment (jfrerich) [inline plugin.json]:
```suggestion
        "footer": "* To report an issue, make a suggestion or a contribution, [check the repository](https://github.com/mattermost/mattermost-plugin-bitbucket)."
```

**Review by jfrerich — CHANGES_REQUESTED** [review]
We should point these to the mattermost org now.

**Comment by hanzei** [comment]
@jfrerich Good catch. Opened https://github.com/mattermost/mattermost-plugin-bitbucket/pull/50 for that.

**Comment by hanzei** [comment]
@DHaussermann This is release is review for review

**Review by jfrerich — APPROVED** [review]

**Comment by hanzei** [comment]
@DHaussermann This release is ready for review

**Comment by hanzei** [comment]
@DHaussermann The release is ready for review

**Comment by DHaussermann** [comment]
@aaronrothschild The code changes added since the initial release test have now been tested and passed.

After doing a quick smoke test there are 2 previously undocumented issues. In my judgment, these would not block the `v1.1.1` release. Please advise. If we're satisfied with solving theses in a future release - we can cut this and move on to the marketplace.  
  
https://github.com/mattermost/mattermost-plugin-bitbucket/issues/73 
https://github.com/mattermost/mattermost-plugin-bitbucket/issues/72

**Comment by aaronrothschild** [comment]
Yes, let's fix thiose in a future release, let's cut it!  :). 🎉

**Pull request merged**